conformance_tests/sysman/test_sysman_memory/src/test_sysman_memory.cpp

```diff
@@ -23,12 +25,23 @@ std::mutex mem_mutex;
 std::condition_variable condition_variable;
 uint32_t ready = 0;
 
+uint32_t get_property_length(char *prop) { return std::strlen(prop); }
+const uint32_t numThreads = 10; 
+
 #ifdef USE_ZESINIT
 class MemoryModuleZesTest : public lzt::ZesSysmanCtsClass {};
 #define MEMORY_TEST MemoryModuleZesTest
+
+class MemoryFirmwareZesTest : public lzt::ZesSysmanCtsClass {};
```

Comment:
I guess new fixture may not be required. already existing fixture can be used (MEMORY_TEST)